From: Yifeng <wuyifeng@nd.com.cn>
Date: Wed, 8 Jun 2022 07:40:00 +0000
Subject: AP9-5555 Disable the keyboard shortcuts about the Chromium bookmark function

Fix version: 1.2.0
1. Disable 'Ctrl + D'
2. Disable 'Ctrl + Shift + D'
3. Disable 'Ctrl + Shift + B'

---
 chrome/android/java/src/org/chromium/chrome/browser/KeyboardShortcuts.java | 13 -------------
 1 file changed, 0 insertion(+), 13 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/KeyboardShortcuts.java b/chrome/android/java/src/org/chromium/chrome/browser/KeyboardShortcuts.java
--- a/chrome/android/java/src/org/chromium/chrome/browser/KeyboardShortcuts.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/KeyboardShortcuts.java
@@ -147,8 +147,6 @@ public class KeyboardShortcuts {
                 context.getString(R.string.keyboard_shortcut_chrome_feature_group_header));
         addShortcut(context, chromeFeatureShortcutGroup, R.string.keyboard_shortcut_open_menu,
                 KeyEvent.KEYCODE_E, KeyEvent.META_ALT_ON);
-        addShortcut(context, chromeFeatureShortcutGroup,
-                R.string.keyboard_shortcut_bookmark_manager, KeyEvent.KEYCODE_B, ctrlShift);
         addShortcut(context, chromeFeatureShortcutGroup, R.string.keyboard_shortcut_history_manager,
                 KeyEvent.KEYCODE_H, KeyEvent.META_CTRL_ON);
         addShortcut(context, chromeFeatureShortcutGroup, R.string.keyboard_shortcut_find_bar,
@@ -167,8 +165,6 @@ public class KeyboardShortcuts {
                 KeyEvent.KEYCODE_R, KeyEvent.META_CTRL_ON);
         addShortcut(context, webpageShortcutGroup, R.string.keyboard_shortcut_reload_no_cache,
                 KeyEvent.KEYCODE_R, ctrlShift);
-        addShortcut(context, webpageShortcutGroup, R.string.keyboard_shortcut_bookmark_page,
-                KeyEvent.KEYCODE_D, KeyEvent.META_CTRL_ON);
         addShortcut(context, webpageShortcutGroup, R.string.keyboard_shortcut_zoom_in,
                 KeyEvent.KEYCODE_EQUALS, KeyEvent.META_CTRL_ON);
         addShortcut(context, webpageShortcutGroup, R.string.keyboard_shortcut_zoom_out,
@@ -308,15 +304,6 @@ public class KeyboardShortcuts {
                     menuOrKeyboardActionController.onMenuOrKeyboardAction(
                             R.id.focus_url_bar, false);
                     return true;
-                case CTRL | SHIFT | KeyEvent.KEYCODE_B:
-                    menuOrKeyboardActionController.onMenuOrKeyboardAction(
-                            R.id.all_bookmarks_menu_id, false);
-                    return true;
-                case KeyEvent.KEYCODE_BOOKMARK:
-                case CTRL | KeyEvent.KEYCODE_D:
-                    menuOrKeyboardActionController.onMenuOrKeyboardAction(
-                            R.id.bookmark_this_page_id, false);
-                    return true;
                 case CTRL | KeyEvent.KEYCODE_H:
                     menuOrKeyboardActionController.onMenuOrKeyboardAction(
                             R.id.open_history_menu_id, false);
--
2.17.1